src/client/search/searchActions.js

```diff
@@ -35,9 +35,10 @@ export const RECEIVE_FEED_RESULTS = 'RECEIVE_FEED_RESULTS';
 export const searchApiForFeed = (searchTerms) =>
 (dispatch) => (
   newPromiseChain()
-  .then(() => (fetchPost('/search', searchTerms)))
-  .then(response => (response.json()))
-  .then(json => dispatch({ type: RECEIVE_FEED_RESULTS, data: json }))
+    .then(() => NProgress.start())
```

Comment:
I'm confused, what's this Nprogress actually doing here :S ? I can't see it being used anywhere or where NProgress even comes from